### PR TITLE
Plugin pytorch docs

### PR DESCRIPTION
**Pull request by igorvalko**

### CODE REVIEW

File: rsts/user/tasktypes/pytorch.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+PyTorch Task Type allows users to run distributed PyTorch training jobs on the Kubernetes cluster via `PyTorch Operator`_.
+
+#######
+ Setup
+#######
+
+In order to build image that is to be eventually submitted to Kubernetes, you'll need to make sure it includes following:
+- pytorch and its dependencies (GPU support, distributed communication backend libs and etc.)
+- flytekit with pytorch extra (``pip install flytekit[pytroch]``)
```
Comment (kumare3) [inline rsts/user/tasktypes/pytorch.rst]: typo

**Comment by kumare3** [comment]
Couple comments

File: rsts/user/tasktypes/pytorch.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @inputs(int_list=[Types.Integer])
+    @outputs(result=Types.Integer)
+    @pytorch_task(
+         workers_count=2,
+         per_replica_cpu_request="500m",
+         per_replica_memory_request="4Gi",
+         per_replica_memory_limit="8Gi",
+         per_replica_gpu_limit="1",
+    )
+    def my_pytorch_job(wf_params, int_list, result):
```
[the remark this answers is not in the document]
Comment (igorvalko) [reply rsts/user/tasktypes/pytorch.rst:37]: I considered it, but it's sort of huge for docs. Once I'll push it to flytesnacks, I'm going to put the link here.

[the remark this answers is not in the document]
Comment (kumare3) [reply rsts/user/tasktypes/pytorch.rst:37]: I guess that is fine. Just that this doc is incorrect for distributed training right now. Reason is, we really do not show how to setup the distributed training context, data loader etc.
I think we should atleast the common boilerplate code here

[the remark this answers is not in the document]
Comment (igorvalko) [reply rsts/user/tasktypes/pytorch.rst]: I've added an excerpt from flytesnacks as agreed previously. Please take a look.

**Comment by kumare3** [comment]
Looks good to me, lets ship it!

**Pull request merged**